docs/README.md

```diff
@@ -160,7 +161,7 @@ Context is created per request and contains following props:
 
 ```js
 bot.use((ctx) => {
-  ctx.telegram             // Telegram client instance
+  ctx.tg                   // Telegram client instance
```

Comment:
We have [public accessor](https://github.com/telegraf/telegraf/blob/develop/core/context.js#L62) for Telegram instance.